### ISSUE TITLE
Craft 4.4: Declaration of verbb\buttonbox\fields\Colours array must be compatible with craft\fields\BaseOptionsField

### ISSUE THREAD

**Issue by tomkiss** [supercool/buttonbox#52]
### Describe the bug

Heads up - I'm testing out [Craft 4.4](https://github.com/craftcms/cms/tree/4.4) and I'm seeing this error with Button Box when attempting to edit a custom field in the CP.

```php
Declaration of verbb\buttonbox\fields\Colours::translatedOptions(bool $encode = false): array 
must be compatible with 
craft\fields\BaseOptionsField::translatedOptions(bool $encode = false, mixed $value = null, ?craft\base\ElementInterface $element = null): array
```

Despite 4.4 still in development, I'm reporting because it seems like there may be a [change on Craft's side](https://github.com/craftcms/cms/pull/12351) that requires this plugin to be updated. See [full 4.4 WIP Changelog here](https://github.com/craftcms/cms/blob/4.4/CHANGELOG-WIP.md).

Please disregard if I'm mistaken! 🙂

### Steps to reproduce

1. Ensure Button Box is installed on a Craft 4.3.X instance with the ButtonBox plugin installed
2. Upgrade to Craft 4.4
3. Attempt to edit a custom field in the CP
4. You should see the error

### Craft CMS version

4.4.x-dev

### Plugin version

4.1

**Comment by tomkiss** [comment]
Yeah, looks like a simple fix - updating the lines in:
- `src/fields/Buttons.php`
- `src/fields/Colours.php`
- `src/fields/TextSize.php`
- `src/fields/Triggers.php`
- `src/fields/Width.php`

From:
```php
protected function translatedOptions(bool $encode = false): array
```

To the new parameters of:
```php
protected function translatedOptions(bool $encode = false, mixed $value = null, ?ElementInterface $element = null): array
```

...appears to fix it. But then perhaps its best to wait to see if 4.4 ships with the issue.

**Comment by engram-design** [comment]
Yep, aware of this one, and will wait for Craft 4.4 to be released!

**Comment by engram-design** [comment]
You can get this early, by running `composer require verbb/buttonbox:"dev-craft-44 as 3.1.1"`

**Comment by jacobgraf** [comment]
Just tried upgrading to Craft 4.4 and get the following error. Any idea how to fix this since 4.4 is in production now? Thanks!

```Declaration of verbb\buttonbox\fields\Buttons::translatedOptions(bool $encode = false): array must be compatible with craft\fields\BaseOptionsField::translatedOptions(bool $encode = false, mixed $value = null, ?craft\base\ElementInterface $element = null): array```

**Comment by engram-design** [comment]
I'll get a Craft 4.4 release out shortly, but @jacobgraf you can use the above command to get the compatible code.

**Comment by engram-design** [comment]
Released in [4.2.0](https://github.com/verbb/buttonbox/releases/tag/4.2.0)
